content/en/users/tutorials/jupyter-datahub-virtual-machine/_index.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Create a Virtual Machine with Jupyter and DataHub"
+type: docs
+weight: 100
+description: >
+  Step by step guide to get a Virtual Machine for Jupyter and DataHub in your
+  cloud provider
+---
+
+## Overview
+
+This tutorial describes how to start a Virtual Machine in the EGI Federation
+that runs a browser-accessible jupyter server with DataHub spaces mounted. This
+setup can be used in the EGI Federation or in any other provider which
+synchronise images from AppDB but is not part of the federation.
+
+## Requirements
+
+This tutorial assumes you have:
+
+- A valid EGI account: learn to can create one in
+  [Check-in](../../aai/check-in/signup).
+- Access to a cloud provider where the
+  [Jupyter DataHub VM is available](https://appdb.egi.eu/store/vappliance/jupyter.datahub.vm).
+  Alternatively, this VM can be run on your computer using a virtualization tool
+  like VirtualBox.
+
+## Create a VM with Jupyter and DataHub
+
+### Step 1: Start your VM
+
+Start your VM on your cloud provider or virtualization tool. You can check
+[the tutorial on how to start a VM](../create-your-first-virtual-machine) to
+learn how to start a VM at EGI's Federated Cloud infrastructure.
+
+This VM does not contain any default credentials, in order to access it you need
+a ssh key. Check
+[this FAQ entry](../../compute/cloud-compute/faq/#how-can-i-inject-my-public-ssh-key-into-the-machine)
+for more information. If you are starting this VM on VirtualBox, you will need
+to pass some valid context for cloud-init, see
+[here how to prepare it](https://superuser.com/a/853957).
+
+The VM image is ready to listen on port `22` for ssh access and port `80` for
+accessing the notebooks interface. Make sure your have those ports open on your
+security groups, otherwise you will not be able to reach the Jupyter notebooks.
+
+Once your instance is ready,
+[assign it a public IP](../../compute/cloud-compute/faq/#how-can-i-assign-a-public-ip-to-my-vm)
+so you can reach it from your computer.
+
+### Step 2: Get your token for the jupyter server
```

Comment:
```suggestion
### Step 2: Get your token for the Jupiter server
```

Reply:
Oh, maybe you meant to add an upper case first letter and made a typo/automatic correction of the y/i